packages/block-directory/src/components/downloadable-block-header/index.js

```diff
@@ -15,7 +15,7 @@ function DownloadableBlockHeader( { icon, title, rating, ratingCount, onClick }
 		<div className="block-directory-downloadable-block-header__row">
 			{
 				icon.match( /\.(jpeg|jpg|gif|png)$/ ) !== null ?
-					<img src={ icon } alt="block icon" /> :
+					<img src={ icon } alt={ sprintf( __( '%s block icon' ), title ) } /> :
```

Comment:
Any string with placeholders needs an accompanying translator comment explaining the placeholder.

Reply:
@swissspidy. Thanks for the heads up. Updated.

Reply:
Thank you! :-)